src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, Input, ElementRef, Renderer} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{class}} {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePorcentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  _value: number;
+  _class: string;
+  min = 0;
+
+  @Input() set class
```

Comment:
Definitely, not only for older browsers but clang is doing weird things to it.

Still, we have to decide if we want this behavior, perhaps use @HostBindings or even go the `striped` attribute (so we won't need to do all this class thing for this component).